spec/02-integration/02-dao/07-ttl_spec.lua

```diff
@@ -28,9 +29,11 @@ helpers.for_each_dao(function(kong_config)
 
       ngx.sleep(1)
 
-      row, err = factory.apis:find {id = api.id}
-      assert.falsy(err)
-      assert.falsy(row)
+      spec_helpers.wait_until(function()
+        row, err = factory.apis:find {id = api.id}
+        assert.falsy(err)
+        return row == nil
```

Comment:
wouldn't it be better to not use asserts in the `wait_until` callback?

`return row == nil and not err` ?

same for below...